CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - **[UPDATE]** Added aria attributes and focus trap on Modals
+- **[FIX]** `Button` do not display children on loading and checked status
```

Comment:
`Button` doesn't display children on `LOADING` and `CHECKED` statuses

Reply:
Done

Reply:
Sorry I'm late, I would say "shouldn't display", else I'm not sure if we're talking about the fix or the problem